src/Java.Interop/Java.Interop/JniRuntime.cs

```diff
@@ -182,11 +183,11 @@ public static void SetCurrent (JniRuntime newCurrent)
 
 		internal    bool                                TrackIDs            {get; private set;}
 		internal    bool                                NewObjectRequired   {get; private set;}
+		internal    CreationOptions                     Options             { get; }
```

Comment:
This shouldn't be needed; just add a `JniAddNativeMethodRegistrationAttributePresent` property, assign it, as is done with `JniVersion`, `InvocationPointer`, etc.  The approach here will just keep the `CreationOptions` instance alive ~forever, "wasting" some amount of memory that would otherwise be collectible.